x-pack/plugins/ml/public/application/contexts/ml/data_source_context.tsx

```diff
@@ -50,14 +50,12 @@ export const DataSourceContextProvider: FC = ({ children }) => {
     sort: false,
   }) as { index: string; savedSearchId: string };
 
-  const getDataViewAndSavedSearchCallback = useCallback(
-    async (ssId: string) => {
-      return await getDataViewAndSavedSearch({
+  const getDataViewAndSavedSearchCb = useCallback(
```

Comment:
nit, we generally don't use `Cb` as shorthand for `Callback`. for consistency could this use `Callback`?

Reply:
I actually think this could be renamed to `const getDataViewAndSavedSearch`. What do you think @jgowdyelastic?

Reply:
Updated here [`30f7dab` (#159625)](https://github.com/elastic/kibana/pull/159625/commits/30f7dabe60780faf434dd3cb0746e773507ee0c7)